bindings/nodejs/tests/closestSoletta.js

```diff
@@ -0,0 +1,19 @@
+var fs = require( "fs" );
+var path = require( "path" );
+var async = require( "async" );
+var dirToCheck;
+
+// Find the closest instance of soletta via its package.json
+function closestSoletta( startDir ) {
+	for ( dirToCheck = startDir; dirToCheck !== "/";
+			dirToCheck = path.resolve( dirToCheck, ".." ) ) {
+		try {
+			if ( JSON.parse( fs.readFileSync(
+					path.join( dirToCheck, "package.json" ), "utf8" ) ).name == "soletta" ) {
+				return dirToCheck;
+			}
+		} catch( errors ) {}
```

Comment:
It makes sense :)

I looked at the rest of the patch and everything looks good to me

:+1: